wpilibc/src/main/native/include/frc/livewindow/LiveWindow.h

```diff
@@ -206,6 +203,9 @@ class LiveWindow {
  private:
   LiveWindow();
 
+  LiveWindow(LiveWindow&&) = default;
+  LiveWindow& operator=(LiveWindow&&) = default;
+
```

Comment:
These shouldn't be private.

Reply:
I don't think we want users to be able to move the instance they get from LiveWindow::GetInstance() out of there though. Perhaps it's best just to delete the copy constructor as was previously done.

Reply:
correct, they should either be public or deleted.